spark-native/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:latest
+
+MAINTAINER Erik Erlandson <eje@redhat.com>
+
+# when the containers are not run w/ uid 0, the uid may not map in
+# /etc/passwd and it may not be possible to modify things like
+# /etc/hosts. nss_wrapper provides an LD_PRELOAD way to modify passwd
+# and hosts.
+RUN yum install -y epel-release tar java && \
+    yum install -y nss_wrapper && \
+    yum clean all
+
+# RUN cd /opt && \
+#     curl https://dist.apache.org/repos/dist/release/spark/spark-2.0.0/spark-2.0.0-bin-hadoop2.7.tgz | \
+#        tar -zx && \
+#    ln -s spark-2.0.0-bin-hadoop2.7 spark
+
+ENV PATH=$PATH:/opt/spark/bin
+ENV SPARK_HOME=/opt/spark
+
+# Add scripts used to configure the image
+COPY scripts /opt/scripts/
+
+COPY spark-distro.tgz /opt/spark/
```

Comment:
what about in-lining the build here and pulling a clone of the repo that has the k8s-native patches?